test/chplcheck/CaseRules.chpl

```diff
@@ -59,4 +59,8 @@ module CaseRules {
     var temp1 = a,
         temp2 = b;
   }
+
+  record testRecord {
+    proc init=(other: testRecord) {}
```

Comment:
Is this going to have unused formal warnings?

Reply:
It sure will, I caught that in my test :D